api/src/main/java/com/datastrato/gravitino/rel/TableChange.java

```diff
@@ -1285,4 +1335,56 @@ public int hashCode() {
       return result;
     }
   }
+
+  /** A TableChange to update the autoIncrement of a field. */
```

Comment:
please add the description about `autoIncrement`,  true is to add autoIncrement, false is to delete autoIncrement.